src/main/java/teammates/common/datatransfer/FeedbackResponseCommentSearchResultBundle.java

```diff
@@ -13,17 +13,27 @@
 
 /**
  * The search result bundle for {@link FeedbackResponseCommentAttributes}.
+ * Mapping details retrieved from {@link teammates.storage.search.FeedbackResponseCommentSearchDocument#fromResults}.
  */
 public class FeedbackResponseCommentSearchResultBundle extends SearchResultBundle {
 
-    public Map<String, List<FeedbackResponseCommentAttributes>> comments = new HashMap<>();
-    public Map<String, List<FeedbackResponseAttributes>> responses = new HashMap<>();
-    public Map<String, List<FeedbackQuestionAttributes>> questions = new HashMap<>();
+    // session name to session map
```

Comment:
Use proper javadoc instead of code comments.
```java
/**
 * Session name to session maps.
 */
public Map<String, FeedbackSessionAttributes> sessions = new HashMap<>();
```